### PR TITLE
Add configuration for https://github.com/probot/stale

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Add configuration for https://github.com/probot/stale

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by markekraus — APPROVED** [review]

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Label to use when marking as stale
+staleLabel: Review - Abandoned
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This PR has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs within 10 days.
```
Comment (SteveL-MSFT) [inline .github/stale.yml]: Instead of "recent activity", I think you should say specifically "activity in the last 30 days".

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 30
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 10
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels: []
+#  - pinned
```
Comment (adityapatwardhan) [inline .github/stale.yml]: Should we consider exempting Issue-Meta and Issue-Discussion?

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: .github/stale.yml (hunk 2 above)
Comment (iSazonov) [reply .github/stale.yml]: and WIP PRs?

**Comment by iSazonov** [comment]
Should we update our contribute docs with the policy?

File: .github/stale.yml (hunk 2 above)
Comment (daxian-dbw) [reply .github/stale.yml]: We will enable the bot only for PRs for now, so `Issue-Meta` and `Issue-Discussion` don't apply.
For `WIP` PRs, I think we should treat them the same way.

File: .github/stale.yml (hunk 1 above)
Comment (daxian-dbw) [reply .github/stale.yml]: Better mention the maintainers in the comment. I suggest adding `/cc @PowerShell/powershell-maintainers` at the end of the comment.

File: .github/stale.yml (hunk 2 above)
Comment (TravisEz13) [reply .github/stale.yml]: the issue labels are for issues, not PRs.  I don't see any reason to exempt them.

**Comment by TravisEz13** [comment]
This is less stringent than our documented process:  https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-requests---abandoned

**Comment by TravisEz13** [comment]
@adityapatwardhan @SteveL-MSFT  Can you update your review?

**Review by adityapatwardhan — APPROVED** [review]

**Review by SteveL-MSFT — APPROVED** [review]

**Pull request merged**